### PR TITLE
Upgrade to webpack 5 

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: config/webpack/plugins/createTreatPlugin.js [hunk 1]
```diff
@@ -17,11 +17,9 @@ const braidThemes = [
 module.exports = ({
   target,
   isProductionBuild,
-  include,
```
Comment (michaeltaranto) [inline config/webpack/plugins/createTreatPlugin.js:20]: This is still being passed through at the [call site](https://github.com/seek-oss/sku/pull/616/files#diff-4e4806abd6e6dec03e7f50c209f69b659d0ceeca5fae03a88e3c40876b6e1de5L193)

File: config/webpack/plugins/sku-webpack-plugin/index.js [hunk 2]
```diff
@@ -163,28 +162,42 @@ class SkuWebpackPlugin {
...
-      compiler.options.resolve.extensions.push('.ts');
-    }
+    if (!compiler.options.resolve.extensions) {
+      compiler.options.resolve.extensions = [
+        '.mjs',
+        '.js',
+        '.json',
+        '.ts',
+        '.tsx',
+      ];
```
Comment (michaeltaranto) [inline config/webpack/plugins/sku-webpack-plugin/index.js:192]: I know this is pretty standard, but would it be better to share this list between the webpack config and here to make sure they are in-sync?

File: config/webpack/webpack.config.js [hunk 3]
```diff
@@ -154,13 +144,15 @@ const makeWebpackConfig = ({
       optimization: {
         nodeEnv: process.env.NODE_ENV,
         minimize: isProductionBuild,
+        minimizer: [new TerserPlugin({ extractComments: false })],
```
Comment (michaeltaranto) [inline config/webpack/webpack.config.js:147]: GIven their api is a little misleading, worth adding a comment that this is to preserve inline license comments?

**Review by michaeltaranto — APPROVED** [review]
Massive piece of work 👏 !

**Review by jahredhope — APPROVED** [review]

**Pull request merged**